src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -1564,7 +1564,7 @@ internal static string GetWinrmPluginShellName()
             Hashtable versionTable = PSVersionInfo.GetPSVersionTable();
```

Comment:
It seems `versionTable` is not used anymore. This line can be removed.

Reply:
Removed.

I see TODO comment - currently we can detect that we use 'Release Tag' and automate the name selection. Make sense?

Reply:
I don't really understand the intention of this comment. Maybe we should open an issue to track it (fix or remove).